torch/csrc/jit/codegen/cuda/kernel_cache.cpp

```diff
@@ -13,6 +13,23 @@ namespace cuda {
 
 namespace {
 
+int getCommonDeviceCUDA(const at::ArrayRef<IValue>& inputs) {
```

Comment:
pls add a comment explaning what it does, and more importantly what the return value means